functions/src/http/index.ts

```diff
@@ -1,7 +1,22 @@
 import type * as express from 'express';
 import { https, logger } from 'firebase-functions';
+import { AddParticipant } from '../db';
+import { IParticipant } from '../models/events';
 
 export const OnHelloWorld = (request: https.Request, response: express.Response) => {
   logger.info("Hello logs!", { structuredData: true });
   response.send("Hello from Firebase!");
 };
+
+
+export const OnJoinEvent = async (data: any, context: https.CallableContext) => {
+  const userId = context.auth!.uid;
+  const participant: IParticipant = {
+    name: data.name,
+    rsvp: {
+      attending: false,
```

Comment:
let's make default true, and they can uncheck if they dont want to participate. It's a weird user experience to first join the party, and then say I also want to participate in secret santa